### PR TITLE
Change ownership and permissions

### PR DESCRIPTION
**Pull request by ahayden**
This PR is part of completing Kristen's offboarding. We'll have a follow up for Xengie, I guess.

Passes pre-commit.

### CODE REVIEW

File: config/prod/TESLA-download-bucket.yaml [hunk 1]
```diff
@@ -4,14 +4,14 @@ template_path: "remote/s3-bucket-v2.j2"
...
-  SynapseIDs: ["1968150", "3382314"]  # Kristen Dang and Xengie Doan
+  SynapseIDs: ["3382314"]  # Xengie Doan
 parameters:
   # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
   Department: "CompOnc"
   # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
   Project: "TESLA"
   # The resource owner
-  OwnerEmail: "kristen.dang@sagebase.org"
+  OwnerEmail: "xengie.doan@sagebase.org"
```
Comment (zaro0508) [inline config/prod/TESLA-download-bucket.yaml:14]: who should be owner now @xdoan ?

Comment (ahayden) [reply config/prod/TESLA-download-bucket.yaml:14]: @xdoan for now, then probably @jaeddy

Comment (brucehoff) [reply config/prod/TESLA-download-bucket.yaml:14]: Since Xengie leaves tomorrow, shouldn't this be someone else?

**Comment by jaeddy** [comment]
Not sure what this resource is. If you need a placeholder account to "babysit" until someone else can more actively maintain, feel free to use mine (it's highly unlikely I'll be using the bucket). Since Kristen left, Justin has been the nominal lead for TESLA — but assigning ownership to him is probably not going to be particularly useful.

**Comment by ahayden** [comment]
Closing as superceded by https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/433

**Pull request closed without merging**